examples/playbook/bigbluebutton.yml

```diff
@@ -0,0 +1,7 @@
+---
+
+- hosts: [ 'bigbluebutton' ]
```

Comment:
```suggestion
- hosts: bigbluebutton
```

Reply:
Maybe the servergroup could be called `bbb_servers` or `bbb_nodes`. But that's a matter of taste.